openraft/src/raft.rs

```diff
@@ -353,14 +355,20 @@ impl<C: RaftTypeConfig, N: RaftNetworkFactory<C>, S: RaftStorage<C>> Raft<C, N,
         turn_to_learner: bool,
     ) -> Result<ClientWriteResponse<C>, ClientWriteError<C>> {
         tracing::info!("change_membership: start to commit joint config");
+        if members.is_empty() {
+            return Err(ClientWriteError::ChangeMembershipError(
+                ChangeMembershipError::EmptyMembership(EmptyMembership {}),
+            ));
+        }
```

Comment:
There is already empty membership check in the `LeaderState::change_membership()`.
This may not be necessary.